### PR TITLE
Config/summary update

### PR DESCRIPTION
**Pull request by zhemingfan**
Update configuration setting to include a parameter allowing users to customize distance by which breakpoints will match the DGV input file.

### CODE REVIEW

File: src/mavis_config/config.json [hunk 1]
```diff
@@ -489,6 +489,11 @@
             "description": "skip the validation stage of the MAVIS pipeline",
             "type": "boolean"
         },
+        "summary.cluster_radius": {
+            "default": 10,
+            "description": "Maximum distance allowed between breakpoint pairs of called SVs and those of known SVs",
```
Comment (creisle) [inline src/mavis_config/config.json]: you may need to be more explicit here, like "Distance used in matching input SVs to reference SVs through clustering"

File: src/mavis_config/config.json [hunk 2]
```diff
@@ -489,6 +489,11 @@
...
+            "type": "integer"
```
Comment (creisle) [inline src/mavis_config/config.json:496]: You should specify a minimum of 0 here so we don't get nonsensical negative inputs

**Review by creisle — CHANGES_REQUESTED** [review]

**Comment by creisle** [comment]
I thought you were adding validate.assembler, is that going to be in a separate PR? also redbeans to validate.aligner enum

**Comment by zhemingfan** [comment]
> I thought you were adding validate.assembler, is that going to be in a separate PR? also redbeans to validate.aligner enum

That'll be in a separate PR.

File: src/mavis_config/config.json (hunk 2 above)
Comment (creisle) [reply src/mavis_config/config.json:496]: I think you may have misunderstood. The default can stay as 10, but you can add a new field "minimum": 0 

see spec docs here https://json-schema.org/understanding-json-schema/reference/numeric.html#range

**Review by creisle — CHANGES_REQUESTED** [review]

**Review by creisle — APPROVED** [review]

**Pull request merged**